### ISSUE TITLE
Unnecessary logic that can be done on sql server side

### ISSUE THREAD

**Issue by zhurlik** [Brest-Java-Course-2018/valentine-ezugu#1]
https://github.com/Brest-Java-Course-2018/valentine-ezugu/blob/8e771ecc72b5e6fcc9893bf0ca30cb05ade324b6/dao/src/main/java/com/epam/brest/course/dao/TruckDaoImpl.java#L101

**Comment by valentine-ezugu** [comment]
yes i can use unique key word sql. Was just practicing as they showed at the java course program. I will change it thank you.
